packages/circuit-ui/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @sumup/circuit-ui
 
+## 4.0.0-next.5
+
+### Patch Changes
+
+- [#1197](https://github.com/sumup-oss/circuit-ui/pull/1197) [`edd6f487`](https://github.com/sumup-oss/circuit-ui/commit/edd6f487ed6ab36d52b50b1a90e4f96a506deb66) Thanks [@robinmetral](https://github.com/robinmetral)! - Hid deprecation warnings on prod and test environments.
```

Comment:
```suggestion
- [#1197](https://github.com/sumup-oss/circuit-ui/pull/1197) [`edd6f487`](https://github.com/sumup-oss/circuit-ui/commit/edd6f487ed6ab36d52b50b1a90e4f96a506deb66) Thanks [@robinmetral](https://github.com/robinmetral)! - Hide deprecation warnings on prod and test environments.
```

Reply:
`Hid` was the past tense to match what we've been using 😛 but it's a bit strange I agree. do you think I should still make it `Hide`?

Reply:
Ooooh, I didn't get that. I don't think I've ever seen 'hid' written anywhere, that's why it looked strange. 

No, let's keep it then.